stable/kommander/requirements.yaml

```diff
@@ -15,3 +15,8 @@ dependencies:
     version: 4.5.*
     repository: https://kubernetes-charts.storage.googleapis.com
     condition: grafana.enabled
+  # Bug with `helm dep update` on local subchart: https://github.com/helm/helm/issues/3742
```

Comment:
You can still add it, i think there is a working example of this in kubefed and nvidia. Please check that.